### ISSUE TITLE
Great Work! 🤩 + Headless CMS Question

### ISSUE THREAD

**Issue by stephenajulu** [Lednerb/bilberry-hugo-theme#237]
Hi, this is great work been looking up and down for a theme like this that supports: linklogging, imagelogging, bloging, videologging and microbloging.
Just saw it today.
Great work. I love it.
Now to the question does it support headless cms e.g Netlify CMS or Forestry?
If not kindly send me details on the front matters on each post type this theme supports i attempt to add Netlify CMS.
The reason i am looking forward to adding or getting this added is because it will be useful when it comes to mobile.
I will be able to easily maintain a public log/microblog of some sorts where i can share ideas, ongoing projects, quick tips, discoveries and more right from my mobile phone.
I really thank you for this. 
Keep up the good work.
Once i get those frontmatter I'll attempt on my own until the work then send a PR to add support for netlify cms. Forestry is abit tricky hence i will try that once netlify works well. But that is if you dont implement it first.
Kindly send me any other info when you can.
Have a great day and thanks again.

**Comment by Lednerb** [comment]
Hey @stephenajulu!
thank you for the nice feedback, I appreciate it very much!


>  Now to the question does it support headless cms e.g Netlify CMS or Forestry?

I'm not into the details of Netlify CMS or Forestry at the moment. I've read about it in the past, but did not use it yet.
It sounds very promising for your usecase and I also think this would be a great addition to this theme.

--------

Regarding the information about the post types: 
I assume you've found the information from the [README.md](https://github.com/Lednerb/bilberry-hugo-theme#Post-Types).

The cool thing about the theme's implementation is, that you can simply use the `hugo new POST_TYPE/ARTICLE_NAME` standard command to setup the next article with the correct post type and also get the necessary frontmatter.

However, the default front-matter config can be found in the [`archetypes` folder](https://github.com/Lednerb/bilberry-hugo-theme/tree/master/archetypes).

Of course there are some additional frontmatter options like [the `original_url`](https://github.com/Lednerb/bilberry-hugo-theme#Reposting-an-Article--Duplicated-Content-SEO) or one for [overwriting the calculated reading time](https://github.com/Lednerb/bilberry-hugo-theme#overwrite-the-calculated-reading-time) etc., but this should all be documented in the `README.md` file.

-------------

Let me know if you have further questions.
I'm looking forward for your PRs! :slightly_smiling_face:

**Comment by Lednerb** [comment]
Today I've tested forestry.

I was able to successfully run a build on their platform.
However, in my opinion it's a non-intuitive way of handling the content and the site and there are some points that have to be configured in much detail and it will still be not as flexible as using the file browser of the users OS.

For example:
Generating a new gallery entry is as simple as:

```
hugo new gallery/my-new-post/_index.md
```

The content can be edited in the `_index.md` file with any editor of choice, also enhanced markdown editors are supported like [Typora](https://typora.io/) or [Mark Text](https://marktext.app/).

Images can simply be added by copying them into the `my-new-post` folder.

Publishing by running `hugo` and deploying the `public` folder.

---------------

In forestry there seems to be no option to copy the image files in the `my-new-post` folder but has to be added via frontmatter (this theme does support this).

--------------

All in all I think it is way more productive to handle these things by oneself, so I wlll not invest any time in supporting third-parties.
However, I'm open for PRs with a detailed step-by-step manual on how to set this up for provider XYZ.
